sdk/tables/azure-data-tables/azure/data/tables/_encoder.py

```diff
@@ -50,6 +50,8 @@ def prepare_value(  # pylint: disable=too-many-return-statements
         if isinstance(value, str):
             return None, value
         if isinstance(value, int):
+            if self._is_int64(value):
+                return EdmType.INT64, str(value)
             return None, value  # TODO: Test what happens if the supplied value exceeds int32.
```

Comment:
The TODO on this line seems incorrect by now, since we just tested if it's int64

Reply:
Removed the TODO comment.